tests/compile-pass/enum.rs

```diff
@@ -1,4 +1,5 @@
-#![feature(lang_items, no_core)]
+// xfail - this makes an unknown error when we try to interpret it in binaryen.
```

Comment:
I made a change to also interpret the compile-pass tests. It would be better to just copy them all into run-pass, but this was quicker for now. Ultimately, I'd like to merge all the compile-pass tests into run-pass, so we don't make any distinction.